apps/sequencer/src/http_handlers/admin.rs

```diff
@@ -131,7 +194,13 @@ pub async fn get_key(
     info!("getting key {} for network {} ...", key, network);
     let result = actix_web::rt::time::timeout(
         Duration::from_secs(7),
-        get_key_from_contract(&sequencer_state.providers, &network, key, decimals),
+        adfs_get_key_from_contract(
+            &sequencer_state.providers,
+            &network,
+            key,
+            decimals,
+            stride as u8,
```

Comment:
Why is this cast here and not `u8` by construction? I see it's `u16` in the `struct` definition.